packages/functional-tests/src/app.ts

```diff
@@ -5,6 +5,7 @@
 
 import * as MRESDK from '@microsoft/mixed-reality-extension-sdk';
 import * as MRERPC from '@microsoft/mixed-reality-extension-sdk/built/rpc';
+import AltspaceVRExtrasTest from './tests/altspacevr-extras-test';

```

Comment:
Wouldn't video-player-test be better? Can always refactor later, once altspacevr-extras has more than one extra in it :)